Fixed issue with geotiff access from protected s3 bucket (#864)
dcs4cop/xcube: CHANGES.md

> ## Changes in 1.1.2

Reviewer: ```suggestion
## Changes in 1.1.2 (in development)
```

### Fixes

> * Fixed issue with geotiff access from protected s3 bucket

Reviewer: ```suggestion
* Fixed issue where geotiff access from a protected s3 bucket was denied (#864)
```

> * Fixed issue where geotiff access from a protected s3 bucket was denied (#864)

Reviewer 2: ```suggestion
* Fixed issue where geotiff access from a protected s3 bucket was denied (#863)
```
